code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -314,12 +324,25 @@
 				new_group.add_turf(src)
 				our_excited_group = excited_group
 			// shares 4/5 of our difference in moles with the atmosphere
-			our_air.share(planetary_mix, 0.25)
+			our_air.share(planetary_mix, 0.8, 0.8)
 			// temperature share with the atmosphere with an inflated heat capacity to simulate faster sharing with a large atmosphere
 			our_air.temperature_share(planetary_mix, OPEN_HEAT_TRANSFER_COEFFICIENT, planetary_mix.temperature_archived, planetary_mix.heat_capacity() * 5)
 			planetary_mix.garbage_collect()
 			PLANET_SHARE_CHECK
 
+	for(var/turf/open/enemy_tile as anything in share_end)
+		var/datum/gas_mixture/enemy_mix = enemy_tile.air
+		archive()
```

Comment:
why are we archiving ourselves each step here? isnt the whole point of sharing to even out the differences from before the sharing started?

Reply:
At this point in the processing, the old archive value doesn't matter, and we just want to allow other tiles to do 100% shares with our tile. You can think of it like we become like them, they become like us (Except space tiles are immutable, so they just reset back to defaults)

If we lie about how much gas we have to give/take, things may end up weird for other non space cases, or if we tried to share with more then one turf at once. That's why I'm doing this, figured it was far enough out of the hot path that it was workable